ci-operator/config/openshift/ironic-rhcos-downloader/openshift-ironic-rhcos-downloader-release-4.5.yaml

```diff
@@ -1,27 +0,0 @@
-base_images:
```

Comment:
did new versions of these files get created somewhere?

Reply:
yes, but I don't know at which point, I haven't double-checked
the problem is that the 4.5 base image doesn't exist and it's blocking the CI
once this is merged I will add the files back and wait for it to be available, but at the moment I don't think we need it